### DISCUSSION TITLE
pretixPOS fields

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3355] [Q&A]
Hello.

Can I add an email mandatory field in the pretixPOS at the box office ?
I checked the demo at pretix.eu and the only fields are "Given name" and "Family name".

Thank you

**Comment by raphaelm** [comment]
You can add fields through Products>Questions, which are also shown in pretixPOS

**Comment by alemao8** [comment]
Can I add this field to be displayed only at the box office and not at Web pretix (on web pretix there is already one email field) ?
What I actually need, is one email field on both Web pretix and box office.
Is this possible ?

**Comment by alemao8** [comment]
Can I display the email field of Contact Information section in pretixPOS ?

**Reply by raphaelm** [reply]
Unfortunately not

**Comment by alemao8** [comment]
Ok I see.
Can I create a field in questions and have it displayed only in box office and not during the web check out process ?

**Reply by wiffbi** [reply]
AFAIK no, but you could create two different products – one for web and one for box office, which share the same quota, but have different questions assigned.
